drivers/dai/intel/dmic/dmic.h

```diff
@@ -0,0 +1,489 @@
+/*
+ * Copyright (c) 2022 Intel Corporation.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef __INTEL_DAI_DRIVER_DMIC_H__
+#define __INTEL_DAI_DRIVER_DMIC_H__
+
+#include <zephyr/sys/util_macro.h>
+
+/* initialization priority */
+#define DMIC_DAI_INIT_PRIORITY 32
+
+/* bit operations macros */
+#define MASK(b_hi, b_lo)	\
```

Comment:
These macros are being repeated pretty often, perhaps its time to put them in sys/util_macro.h next to the similarly useful BIT() and BIT_MASK() macros